test/models/v3/operation.spec.ts

```diff
@@ -150,6 +150,20 @@ describe('Operation model', function() {
     });
   });
 
+  describe('.reply()', function() {
```

Comment:
In this file there're already tests for this method. Remove one of them.

Reply:
done